docs/metrics.md

```diff
@@ -310,7 +310,7 @@ print(detailed_evaluation)
 
 ## General Text Generation Metrics
 
-The general text generation metrics apply to a broad set of text generation tasks. These metrics don't compare to any groundtruths and don't require any sort of context. The metrics are evaluated purely based on the predicted text.
+The general text generation metrics apply to a broad set of text generation tasks. These metrics don't compare to any groundtruths and don't require any contexts. The metrics are evaluated purely based on the predicted text.
```

Comment:
```suggestion
The general text generation metrics apply to a broad set of text generation tasks. These metrics don't compare to any groundtruths and don't require any additional context. The metrics are evaluated purely based on the predicted text.
```

Reply:
using the plural of "context" throughout these docs seems grammatically incorrect to me

Reply:
I changed the one you commented on above and I also changed "contradicts the contexts" to "contradicts any context", but for the most part the use of contexts seems correct. I looked at discussion online of how to use context vs contexts and I also copy and pasted a few sections into chatgpt to ask what was correct. In those sections, it said that the use of contexts was fine, and when I changed to context, it corrected it back to contexts. 

Was there any other spot in the docs where you felt the use of contexts was incorrect?